pipelines/setup.yml

```diff
@@ -37,7 +37,7 @@ jobs:
   - name: self-update
     plan:
       - get: paas-release-ci
-        trigger: {{auto_trigger}}
+        trigger: true
```

Comment:
I'm concerned that this change makes it inconsistent with paas-cf where pipelines on dev deployments don't auto-trigger.